.travis.yml

```diff
@@ -19,6 +19,7 @@ before_script:
 script:
   - yarn lint
   - yarn build
+  - yarn test
```

Comment:
I would suggest to put `yarn test` before `yarn build` since it should go from the fastest tot he slowest and I think that the building task is the slowest one.

Reply:
Makes sense, I'll change the order